assessment/models.py

```diff
@@ -88,6 +88,14 @@ class InteractiveQuiz(AssessmentTool):
     duration_in_minutes = models.IntegerField(null=False)
     total_points = models.IntegerField(null=False)
 
+    def get_tool_data(self) -> dict:
+        tool_base_data = super().get_tool_data()
+        tool_base_data['additional_info'] = {
+            'duration_in_minutes': self.duration_in_minutes,
+            'total_points': self.total_points
```

Comment:
Hello there, Indi. Should the total number of points be revealed to the assessee during the assessment event? I believe it is preferable to limit the information to the assessors. Please let me know what you think:)

Reply:
Okay, thank you for your suggestion Jo, I think I agree with you. I'll go change that :)